### PR TITLE
Ypierson/function summary

### PR DESCRIPTION
**Pull request by Ypierson**
### New Content Checklist

- [ ] New content/chapter is in an Rmd file with [this kind of format and headers](https://github.com/jhudsl/OTTR_Template/blob/main/02-chapter_of_course.Rmd).

- [ ] New content/chapter contains [Learning Objectives and are in the correct format](https://github.com/jhudsl/OTTR_Template/wiki/Setting-up-images-and-graphics#learning-objectives-formatting).

- [ ] [Bookdown successfully re-renders and any new content files have been added to the _bookdown.yml](https://github.com/jhudsl/OTTR_Template/wiki/Publishing-with-Bookdown).

- [ ] Spell check runs successfully in [Github actions style-n-check](https://github.com/jhudsl/OTTR_Template/wiki/How-to-set-up-and-customize-GitHub-actions-robots#spell-check)).

- [ ] Any newly necessary packages that are needed have been added to the [Dockerfile and image](https://github.com/jhudsl/OTTR_Template/wiki/Using-Docker#adding-packages-to-the-dockerfile).

- [ ] Images are in the [correct format for rendering](https://github.com/jhudsl/OTTR_Template/wiki/Setting-up-images-and-graphics#adding-images-and-graphics-in-text).

- [ ] Every new image has [alt text and is in a Google Slide](https://github.com/jhudsl/OTTR_Template/wiki/Setting-up-images-and-graphics#accessibility).

- [ ] Each slide is described in the notes of the slide so learners relying on a screen reader can access the content. See https://lastcallmedia.com/blog/accessible-comics for more guidance on this.

- [ ] The color palette choices of the slide are contrasted in a way that is friendly to those with color vision deficiencies.
You can check this using [Color Oracle](https://colororacle.org/).

### CODE REVIEW

**Comment by cansavvy** [comment]
@Ypierson let me know when you are ready for me to review this! Thanks! Looking good!

File: 02_getting_data_02_objects.Rmd [hunk 1]
```diff
@@ -295,3 +295,25 @@ For now, it's not important to understand the details of that last paragraph, bu
...
+- Can coerce a existing vector to be of a particular class. So for example, 
+- `as.factor()` would try to make the vector a factor. 
+                                                  
+`class()` - Can tell us what kind of data is being stored.
+
+`concatenate` - Can link pieces of information together.
+
+`operators` - The addition, subtraction, multiplication, division, 
+and exponentiation operators in R are `+`, `-`, `*`, `/`, and `^` respectively.
+
```
Comment (cansavvy) [inline 02_getting_data_02_objects.Rmd:318]:
```suggestion
</div>
```

File: 02_getting_data_03_logicals.Rmd [hunk 2]
```diff
@@ -228,3 +228,16 @@ Here, the sum of the `meets_indiv_cut` is 4. When you sum a logical, R returns t
...
+- `!`: the "not" operator
+- `&`: the "and" operator
+- `|`: the "or" operator (_Shift_ + _backslash(\)_ )
+- `==`: the "equals" operator
+- `!=`: the "not equal" operator
+- `>`: the "greater than" operator
+- `>=`: the "greater than or equal to" operator
+- `<`: the "less than" operator
+- `<=`: the "less than or equal to" operator
+- `%in%`: the "contained in" operator
```
Comment (cansavvy) [inline 02_getting_data_03_logicals.Rmd:243]:
```suggestion
- `%in%`: the "contained in" operator
</div>
```

File: 02_getting_data_04_dataframes.Rmd [hunk 3]
```diff
@@ -162,3 +162,17 @@ According to the people who develop the tidyverse:
...
+- `Environment- where you can see your data frame.`
+- `head()`- to see the first 6 rows of a data frame.`
+- `tail()`- to see the last 6 rows of a data frame.`
+- `names()`- to see the column names of the data frame.`
+- `nrow()`- to see how many rows are in the data frame`
+- `ncol()`- to see how many columns are in the data frame.`
+- `summary()`- function gives us a summary of each variable in the data frame.`
+- `str()`- function gives us information about the structure and variables.`
+- `matrix - is like a data frame but it needs to contain all the same data type.`
+- `tibble- similar to a data frame. It is the tidyverse brand of data frame.`
```
Comment (cansavvy) [inline 02_getting_data_04_dataframes.Rmd:178]:
```suggestion
- `tibble- similar to a data frame. It is the tidyverse brand of data frame.`
</div>

```

File: 02_getting_data_04_dataframes.Rmd [hunk 4]
```diff
@@ -162,3 +162,17 @@ According to the people who develop the tidyverse:
 > Tibbles are data.frames that are lazy and surly: they do less (i.e. they don’t change variable names or types, and don’t do partial matching) and complain more (e.g. when a variable does not exist).
 
 They are similar enough, that for most instances, especially when you are working with tidyverse functions, you can treat them the same. However if you get a peculiar error when trying to use a tibble with a non-tidyverse function, you may need to use the `as.data.frame()` function to coerce your tibble to a data frame. But you shouldn't encounter this problem too often. 
+
+<div class = "dictionary">
+- `observations`- are also called rows.`
```
Comment (cansavvy) [inline 02_getting_data_04_dataframes.Rmd:167]:
```suggestion
- `observations`- are also called rows.`
```
```suggestion
- `observations`- are also called rows.
```

Comment (cansavvy) [reply 02_getting_data_04_dataframes.Rmd:167]: Look out for those stray back ticks. You'll want them to be in pairs.

File: Dictionary.R [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+- `&`: the "and" operator
+- `|`: the "or" operator (_Shift_ + _backslash(\)_ )
+- `==`: the "equals" operator
+- `!=`: the "not equal" operator
+- `>`: the "greater than" operator
+- `>=`: the "greater than or equal to" operator
+- `<`: the "less than" operator
+- `<=`: the "less than or equal to" operator
+- `%in%`: the "contained in" operator
+
```
Comment (cansavvy) [inline Dictionary.R]:
```suggestion
</div>
```

File: 00_intro_05_working_on-offline.Rmd [hunk 6]
```diff
@@ -197,3 +197,14 @@ This will place your cursor on the file name which will now be editable. If you
...
+
+<div class = "dictionary">
+- `Cloud storage` - `refers to data or document storage on the Internet 
+rather than on your personal computer.`
+- `Cloud` - `storage that is similar to having an external hard drive 
+(portable storage device).`
+- `Local` - `it isn't on the cloud but instead something stored 
+on your computer or other device.`
+- `Cloud computing` - `involves applications and software that run on shared data centers rather 
+than running on the computer sitting in front of you.`
```
Comment (cansavvy) [inline 00_intro_05_working_on-offline.Rmd:210]:
```suggestion
than running on the computer sitting in front of you.`
</div>
```

File: 00_intro_05_working_on-offline.Rmd [hunk 7]
```diff
@@ -197,3 +197,14 @@ This will place your cursor on the file name which will now be editable. If you
...
 
 Usually it is a good idea to leave the file extension the same even if you rename a file. The extension is all of the characters after the period that tells your computer what type of file it is - a code file, a data file, a text file, or something else. For example if your file is named "file.png" then the extension is ".png". So you might change the file name to something like "new_file.png" but you would want to leave the ".png" at the end.
+
+
+<div class = "dictionary">
+- `Cloud storage` - `refers to data or document storage on the Internet 
+rather than on your personal computer.`
+- `Cloud` - `storage that is similar to having an external hard drive 
+(portable storage device).`
+- `Local` - `it isn't on the cloud but instead something stored
```
Comment (cansavvy) [inline 00_intro_05_working_on-offline.Rmd:207]:
```suggestion
- `Local` - it isn't on the cloud but instead something stored. 
```

File: 00_intro_05_working_on-offline.Rmd [hunk 8]
```diff
@@ -197,3 +197,14 @@ This will place your cursor on the file name which will now be editable. If you
...
+
+
+<div class = "dictionary">
+- `Cloud storage` - `refers to data or document storage on the Internet 
+rather than on your personal computer.`
+- `Cloud` - `storage that is similar to having an external hard drive 
+(portable storage device).`
+- `Local` - `it isn't on the cloud but instead something stored 
+on your computer or other device.`
+- `Cloud computing` - `involves applications and software that run on shared data centers rather
```
Comment (cansavvy) [inline 00_intro_05_working_on-offline.Rmd:209]:
```suggestion
- `Cloud computing` - involves applications and software that run on shared data centers rather 
```

**Review by cansavvy — COMMENTED** [review]
@Ypierson This is a great start! Super excited for this! We talked about this on our meeting, but wherever you have a start `<div>` tag you'll also need a end `</div>` tag. 

Also look out for backticks. They need to come in pairs because markdown uses them to highlight stuff. 

The content though (which is the harder part) looks great!

**Comment by cansavvy** [comment]
@davonperson if you can review this before our meeting tomorrow, that would be great! Thanks!

**Review by davonperson — APPROVED** [review]

**Review by davonperson — APPROVED** [review]

File: Dictionary.Rmd [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+
+`concatenate` - Can link pieces of information together.
+
+`operators` - The addition, subtraction, multiplication, division, 
+and exponentiation operators in R are `+`, `-`, `*`, `/`, and `^` respectively.
+
+
+                                                  
+                                                  
+TODO: finish up this chapter's functions
```
Comment (cansavvy) [inline Dictionary.Rmd:25]: Hey @Ypierson Did you get to this todo yet?

File: Dictionary.Rmd [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+- `&`: the "and" operator
+- `|`: the "or" operator (_Shift_ + _backslash(\)_ )
+- `==`: the "equals" operator
+- `!=`: the "not equal" operator
+- `>`: the "greater than" operator
+- `>=`: the "greater than or equal to" operator
+- `<`: the "less than" operator
+- `<=`: the "less than or equal to" operator
+- `%in%`: the "contained in" operator
+
```
Comment (cansavvy) [inline Dictionary.Rmd:39]:
```suggestion
</div>
```

File: Dictionary.Rmd [hunk 11]
```diff
@@ -0,0 +1,224 @@
...
+<div class = "dictionary">
+- `Cloud storage` - `refers to data or document storage on the Internet 
+rather than on your personal computer.`
+- `Cloud` - `storage that is similar to having an external hard drive 
+(portable storage device).`
+- `Local` - `it isn't on the cloud but instead something stored 
+on your computer or other device.`
+- `Cloud computing` - `involves applications and software that run on shared data centers rather 
+than running on the computer sitting in front of you.`
+
```
Comment (cansavvy) [inline Dictionary.Rmd:62]:
```suggestion
</div>
```

File: Dictionary.Rmd [hunk 12]
```diff
@@ -0,0 +1,224 @@
...
+<div class = "dictionary">
+  -06 ?
```
Comment (cansavvy) [inline Dictionary.Rmd:64]: @Ypierson this looks like it might be dangling, can you wrap this part up?

File: Dictionary.Rmd [hunk 13]
```diff
@@ -0,0 +1,224 @@
...
+  
+  <div class = "dictionary">
+  `SMART Goals` - 
+  - **S**pecific - goals need to be simple and succinct. Elaborate goals or goals that are vague are too hard to track or understand.
+- **M**easurable - goals need to be something you can measure. If you are not able to measure it you won't be able to know if you've met your goal or not.
+- **A**chieveable - to set ourselves up for success, we need to make goals that are actually reasonable.
+- **R**elevant - goals need to be something related to what we'd like to achieve.
+- **T**ime bound - goals need to have a set amount of time that we can accomplish them in.
+
```
Comment (cansavvy) [inline Dictionary.Rmd:73]:
```suggestion
</div>
```

File: Dictionary.Rmd [hunk 14]
```diff
@@ -0,0 +1,224 @@
...
+<div class = "dictionary">
+- **Form a question** - What is the question you hope to answer with data?
+- **Get the data** - To answer this question you'll need data!
+  - **Clean the data** - Datasets are almost never ready to analyze from the get go. As a data scientist, we'll need to do some cleaning and set up steps to get the data where we need it before we can really dig into it.
+- **Explore and plot the data** - After your data is clean, you'll want to do some initial exploration. What do these data look like? Plots are a great way to visual your data and start to get an idea of how the data might relate to your initial question.
+- **Get stats** - You can use statistics and models to try to use your data to get to the bottom of your question.
+- **Share results** - Given everything you are seeing with your data, how do we interpret this? What's our conclusion when it comes to our initial question?
+
```
Comment (cansavvy) [inline Dictionary.Rmd:81]:
```suggestion
</div>
```

File: Dictionary.Rmd [hunk 15]
```diff
@@ -0,0 +1,224 @@
...
+ <div class = "dictionary">
+ - `Descriptive`- `is to understand the components of a data set, describe what they are,
+ and explain that description to others who might want to understand the data.`
+ - `Exploratory` - `find unknown relationships between the different variables you have measured in your data set.`
+ - `Inferential` - `to use a small sample of data to say something about what would happen if we collected more data.`
+ - `Predictive` - `to use data from a large collection to predict values for new individuals.`
+
```
Comment (cansavvy) [inline Dictionary.Rmd:88]:
```suggestion
 </div>
```

File: Dictionary.Rmd [hunk 16]
```diff
@@ -0,0 +1,224 @@
...
+ - `Inferential` - `to use a small sample of data to say something about what would happen if we collected more data.`
+ - `Predictive` - `to use data from a large collection to predict values for new individuals.`
+ 
+ 02_getting data_01
+ <div class = "dictionary">
+ - `Scripting` - where you write your code
+ - `Console` - where your code runs
+ - `Environment` - where you can see what objects have been created during your analysis
+ - `Files` - where you can see all the files that are part of your project
+
```
Comment (cansavvy) [inline Dictionary.Rmd:95]:
```suggestion
 </div>
```

File: Dictionary.Rmd [hunk 17]
```diff
@@ -0,0 +1,224 @@
...
+  `mv` - command for moving and renaming files.
+  `file.rename()` - in R command for moving and renaming files. 
+  `cp` - command for copying and renaming files.
+  `file.copy()` - in R command for copying and renaming files.
+   `rm` - command for deleting files.
+   `file.remove()` - in R command for deleting files.
+   `ls` - command to list the contents of a directory for verifying what files and folders are present.
+   `file.exists()` - in R command to list the contents of a directory for verifying what files and folders are present.
+   file path - The series of folders it takes to get to the file 
+
```
Comment (cansavvy) [inline Dictionary.Rmd:114]:
```suggestion
   </div>
```

File: Dictionary.Rmd [hunk 18]
```diff
@@ -0,0 +1,224 @@
...
+   `file.exists()` - in R command to list the contents of a directory for verifying what files and folders are present.
+   file path - The series of folders it takes to get to the file 
+   
+   02_getting data_08
+ - packages - is a collection of functions, data, 
+   and code conveniently provided in a nice, complete format for you.
+ - library - is the place where the package is located on your computer.
+ - repositories - is central location where many developed packages are located and available for download.
+ - dependencies - packages required by other packages to be loaded first.
+ -
```
Comment (cansavvy) [inline Dictionary.Rmd:121]:
```suggestion
</div>
```

File: Dictionary.Rmd [hunk 19]
```diff
@@ -0,0 +1,224 @@
...
+ 
+ 02_getting data_10
+ - Comma-separated values (CSV) - plain-text files where all the important information
+ in the file is represented by text.
+ - `write_csv()`-  write a  CSV.
+ - `write_tsv()` - write a TSV
+ - `write_rds()` - write a RDS
+ - `saveRDS()`- To save a serialized R object file
+ - `readRDS()` - To read a serialized object file into R
+
```
Comment (cansavvy) [inline Dictionary.Rmd:131]:
```suggestion
 </div>
```

File: Dictionary.Rmd [hunk 20]
```diff
@@ -0,0 +1,224 @@
...
+ - `write_rds()` - write a RDS
+ - `saveRDS()`- To save a serialized R object file
+ - `readRDS()` - To read a serialized object file into R
+ 
+ 02_getting data_11
+- Private data - datasets to which a limited number of people or groups have access.
+- public data - freely-available data 
+- open data - after registering you'll have access to many different types of datasets.
+- Government Data - data sets cover topics from education and student loan debt to climate and weather
+
```
Comment (cansavvy) [inline Dictionary.Rmd:137]:
```suggestion
</div>
```

File: Dictionary.Rmd [hunk 21]
```diff
@@ -0,0 +1,224 @@
...
+02_getting data_13
+- Google Sheets - for holding data
+- Google Docs - for writing up non-code documents
+- spreadsheet - is a type of document where data are stored in rows and columns of a grid
+- To do this open another new tab by holding _ctrl_ and pressing _t_
+
```
Comment (cansavvy) [inline Dictionary.Rmd:143]:
```suggestion
</div>
```

**Comment by cansavvy** [comment]
@Ypierson I made a few edits but this was great and I'm going to merge it!

**Pull request merged**